### PR TITLE
Timeout release 1.5hr

### PR DESCRIPTION
**Pull request by Mitulsharma123**
timeout added as 1.5 hours for release pipeline

### CODE REVIEW

**Comment by Allda** [comment]
@Mitulsharma123 I can help you with the rebase but first can you provide more details on this change? The current timeout is set to 2 hours. Is it really intentional to decrease to 1.5 hours?

**Comment by Mitulsharma123** [comment]
@Allda followed what you've commented in https://github.com/redhat-openshift-ecosystem/operator-pipelines/pull/254#issuecomment-1043216430

File: ansible/roles/operator-pipeline/tasks/webhook-hosted-pipeline-trigger.yml [hunk 1]
```diff
@@ -81,7 +81,7 @@
                 metadata:
                   generateName: operator-hosted-pipeline-run
                 spec:
-                  timeout: "2h"
+                  timeout: "1.5h"
```
Comment (amisstea) [inline ansible/roles/operator-pipeline/tasks/webhook-hosted-pipeline-trigger.yml:84]: This is changing the overall timeout of the hosted pipeline which should stay at `2h`. Please define the timeout in the release pipeline trigger instead.

File: README.md [hunk 2]
```diff
@@ -3,6 +3,8 @@
 Red Hat OpenShift pipelines for certifying ISV Operator Bundles.
 
 ## Getting Started
+=======
+**To run any of the pipelines for the first time, multiple cluster resources have to be created.**
```
Comment (amisstea) [inline README.md:7]: You have a few merged git conflicts scattered around. Please rebase/fix your branch to remove the dupe commit.

**Review by amisstea — CHANGES_REQUESTED** [review]

**Comment by Mitulsharma123** [comment]
@amisstea I've removed the conflict part, let me know if there is a still issue ? let me know, where can i ping you outside github.

**Comment by amisstea** [comment]
@Mitulsharma123 something didn't go according to plan with your rebasing. There are even more commits now and the relevant changes were omitted. Please reach out on gchat if you need assistance.

**Comment by amisstea** [comment]
Dupe of #266. Closing.

**Pull request closed without merging**